tests/test_authentication/test_authentication.py

```diff
@@ -11,6 +11,37 @@
 
 class TestAuthentication(BaseTestCase):
     def test_invalid_token_returns_error_message(self):
+        """
+        Test that an error message is returned when a user
+        has an invalid token
+        """
+        headers = {"Authorization": "Bearer" + " " + "INVALID_TOKEN"}  # noqa E501
+        response = self.app_test.post(
+            '/mrm?query=' + office_mutation_query,
+            headers=headers)
+        self.assertIn("Invalid token. Please Provide a valid token!", str(response.data)) # noqa E501
+
+    def test_no_token(self):
```

Comment:
Use a more descriptive function name for your test functions. Compare how the function in line 13 has been named

Reply:
@Bonifase I have implemented that.